backend/src/schema/resolvers/follow.spec.js

```diff
@@ -41,44 +70,57 @@ describe('follow', () => {
   describe('follow user', () => {
     describe('unauthenticated follow', () => {
       it('throws authorization error', async () => {
-        const client = new GraphQLClient(host)
-        await expect(client.request(mutationFollowUser('u2'))).rejects.toThrow('Not Authorised')
+        authenticatedUser = null
+        const { errors } = await mutate({
+          mutation: mutationFollowUser('u2'),
```

Comment:
```suggestion
          mutation: mutationFollowUser, variables
```
Make the `mutationFollowUser` a `const` instead of a method. The string interpolation originates from a time when we weren't familiar enough with GraphQL and we did not know how to use `variables`.

The pattern which I happen to love is to mutate a global `variables` object in a `beforeEach` of a scope. Then pass these `variables` into the call of `mutate` or `query`. In my opinion that's a good way to avoid false negatives.

Reply:
done